web/static/css/_forms.scss

```diff
@@ -60,42 +60,29 @@ label {
     top: 50%;
     margin: 0 15%;
     &.csv:before {
-      content: 'Drop your CSV file here, or click to browse';
```

Comment:
Let's leave the content: '...' commented in the css and a note to see I18nStyle.jsx

Reply:
OK! Done!